capreolus/searcher/tests/test_searcher.py

```diff
@@ -20,6 +20,24 @@ def test_searcher_runnable(tmpdir_as_cache, tmpdir, dummy_index, searcher_name):
     assert os.path.exists(os.path.join(output_dir, "done"))
 
 
+@pytest.mark.parametrize("searcher_name", searchers)
+def test_searcher_query(tmpdir_as_cache, tmpdir, dummy_index, searcher_name):
+    topics_fn = DummyBenchmark.topic_file
+    query = list(load_trec_topics(topics_fn)["title"].values())[0]
+    nhits = 1
+    searcher = Searcher.create(searcher_name, config={"hits": nhits}, provide={"index": dummy_index})
+    results = searcher.query(query)
+    if searcher_name == "SPL":
```

Comment:
will need to delete this after [this line](https://github.com/castorini/anserini/blob/a5ba12681442c23a7715c54cef9579b6ba2f9834/src/main/java/io/anserini/search/SearchArgs.java#L175-L177) is updated in next released version;
now calling -spl when using Anserini.SearchCollection raise exception `option "-spl" cannot be used with the option(s) [bm25, -qld, -qljm, -inl2, -spl, -f2exp, -f2log]this`